pages/1.12/release-notes/1.12.1/index.md

```diff
@@ -0,0 +1,169 @@
+
+    However, you should not configure the LOG_TO_SYSLOG environment variable to one (1), which uses the `systemd` logging facility. Setting this environment variable to one writes information to the `syslog` facility inside the container. The container and logged information is destroyed when the GC operation is done.
+
+## GUI
+- COPS-3968, DCOS-43897 -  Service labels and environment variable are rendered properly in the DC/OS UI when using Firefox as your web browser.
+
+## Marathon
+- COPS-3554 - This release introduces a watcher loop process to monitor and, if necessary, re-register the Marathon leader after reelection.
+- COPS-3585 - In previous releases, a deadlock or race condition might prevent one or more nodes in a cluster from generating a routing table that forwards network traffic through Marathon load balancing properly. Problems with routing tables and network connectivitity can lead to the following issues:
+    - Incomplete network overlay configuration on certain nodes
+    - Incomplete VIP/IPVS/L4LB configuration on certain nodes
+    - DNS records that are missing on certain nodes
+You can restart the `systemd` process on the nodes affected to restore proper netowrk connectivity. 
+- COPS-3593, DCOS_OSS-4193 - In previous releases, you might have services that are managed by Marathon unable to restart if the container crashes or under certain DNS failure conditions. For example,restarting services might fail if the first ZooKeeper node or first DC/OS master is unreachable. 
+
+    Because this problem affects high availability for Marathon, a workaround (ping zk-1) was introduced for DC/OS 1.11.5 and 1.11.6 to address the issue. In this release, the underlying issue is resolved and you can safely remove the workaround if you have it deployed. For background information about the issue and the steps to remove the workaround, see [Removing the patch for Marathon failing to start if the first DC/OS is not available](https://mesosphere-community.force.com/s/article/Critical-Issue-Marathon-MSPH-2018-0004).
+- COPS-3907, DCOS-44502, DCOS-46093 - You can add and edit environment variables for Marathon-LB and Marathon app definitions by using configuration files or by modifying settings in the DC/OS web-based UI.
+- MARATHON-8413 - Versioning information for apps and pods in Java9 includes a timestamp in nanoseconds if the `Instant.now()` function is called. Previously, nanoseconds were not included in the version information format that was retreived for an applications and pods.
+
+## Mesos
+- DCOS_OSS-4607 - A new heartbeat operation periodically validates successful communication between the agent and task executor. The heartbeat helps prevent any network intermediaries from interfering with agent-to-executor communication when the agent-to-executor connection sits idle for an extended period of time.
```

Comment:
Move this entry and place it after DCOS-46006. This is to maintain the ordering hierarchy. 
Example: List all tickets that begin with `DCOS-number` then list the remaining tickets that begin with `DCOS_OSS-number`

- DCOS-number
- DCOS_OSS-number

Reply:
Done